chartfx-dataset/src/main/java/de/gsi/dataset/serializer/spi/iobuffer/IoBufferSerialiser.java

```diff
@@ -86,91 +85,70 @@ public IoBufferSerialiser(final IoBuffer buffer) {
                 (obj, field) -> {
                     final Map<?, ?> retVal = (Map<?, ?>) field.getField().get(obj);
                     BinarySerialiser.put(ioBuffer, field.getFieldName(), retVal);
-                }, Map.class));
+                },
+                Map.class));
 
         FieldDataSetHelper.register(this, ioBuffer);
 
         // addClassDefinition(null, StringHashMapList.class);
-
-        if (LOGGER.isDebugEnabled()) {
-            LOGGER.atDebug().addArgument(AbstractSerialiser.class.getName()).addArgument(knownClasses().size())
-                    .log("{} added {} known classes");
-        }
     }
 
-    protected void deserialise(final Object obj, final FieldHeader fieldRoot,
-            final ClassFieldDescription classFieldDescription, final int recursionDepth) throws IllegalAccessException {
+    protected void deserialise(final Object obj, final FieldHeader fieldRoot, final ClassFieldDescription classFieldDescription, final int recursionDepth) throws IllegalAccessException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'deserialise(Object, FieldHeader, ClassFieldDescription, int)' has an NPath complexity of 294](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5640920)